CONTRIBUTING.rst

```diff
@@ -53,11 +53,25 @@ that. On Mac and Linux, GCC is required, while Visual C++ 14.0 is necessary for
 
        git clone git@github.com:LCAV/pyroomacoustics.git
 
-2. Compile locally
+2. Install a few pre-requisites
+
+    .. code-block:: shell
+
+        pip install numpy Cython pybind11
+
+3. Install Eigen library. On macOS, you can find installation instruction here:
```

Comment:
Eigen is included in the repository as a sub-module. It should be possible to get it via a recursive clone.

Reply:
Ok didn't know about recursive clone! So I installed Eigen and created the symbolic link. The recursive clone approach would be to run the following?
```
git clone --recursive git@github.com:LCAV/pyroomacoustics.git
```